rebasehelper/specfile.py

```diff
@@ -116,11 +116,13 @@ class SpecFile:
     """Class representing a SPEC file. Be aware that using SpecFile
     modifies RPM macros in global context."""
 
-    def __init__(self, path: str, sources_location: str = '', predefined_macros: Optional[Dict[str, str]] = None):
+    def __init__(self, path: str, sources_location: str = '', predefined_macros: Optional[Dict[str, str]] = None,
```

Comment:
Could you please cover this change in `tests/public_api/test_specfile.py`?

Reply:
Sure thing, done.